client/gutenberg/editor/controller.js

```diff
@@ -3,14 +3,28 @@
  * External dependencies
  */
 import React from 'react';
+import { startsWith } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import GutenbergEditor from 'gutenberg/editor/main';
 
+function determinePostType( context ) {
+	if ( startsWith( context.path, '/gutenberg/post' ) ) {
```

Comment:
should be able to use `String.prototype.startsWith` now as of our inclusion of the core-js polyfills

```js
if ( context.path.startsWith( '/gutenberg/post' ) ) {
	…
}
```

Reply:
I think we're missing the important `/` in this and the next example. do we really want to return `post` for `/gutenberg/posts-are-for-losers/`